README.md

```diff
@@ -21,129 +22,64 @@ This allows you to build anything from simple websites to enterprise-grade intra
 Volto exposes all these features and communicates with Plone via its [REST API](https://github.com/plone/plone.restapi).
 
 Volto features the Pastanaga editor, a modern block-based content layout editor.
-It is extensible and customizable, so you can adapt the default blocks provided to match your requirements, or build new ones to cover them.
+It is extensible and customizable, so you can adapt the provided default blocks to meet your requirements, or build new ones.
 
 Volto is extensible using add-ons.
 You can build your own or choose from the community released ones:
 
 - [Volto Add-ons in NPM](https://www.npmjs.com/search?q=keywords%3Avolto-addon%2Cvolto)
 - [Volto Awesome](https://github.com/collective/awesome-volto)
 
+
 ## Demo
 
-You can try a Volto online demo at [https://6.demo.plone.org/](https://6.demo.plone.org/)
+You can try a Volto online demo at [https://demo.plone.org/](https://demo.plone.org/).
+
 
 ## Create a Volto project
 
 To start a new project using Volto, follow the [Plone installation documentation](https://6.docs.plone.org/install/install-from-packages.html).
 
-## Documentation
-
-You can find the latest documentation at [https://6.docs.plone.org/](https://6.docs.plone.org/volto/index.html).
 
-For links to trainings and videos, see [Other learning resources](https://6.docs.plone.org/volto/getting-started/others.html).
+## Contributing to Volto
```

Comment:
@stevepiercy I think the contributing section belongs further down, in its current location. My rationale is that the contributing section is only relevant to people who want to contribute, but other information like the docs and compatible versions is relevant to anyone using volto.

Reply:
Yeah, that makes more sense in the greater scheme of things. I moved it back in d0ba23c7f95429a1f57013bef6dad41711936d32